releng/buildpacks/auditlog-client/src/main/java/com/sap/auditlog/client/auditlogs/Log.java

```diff
@@ -0,0 +1,128 @@
+package com.sap.auditlog.client.auditlogs;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Log {
+
+  @JsonProperty("message_uuid")
+  private String uuid;
```

Comment:
Nope, the parameter is required, it isn't autogenerated by auditlog service. According to the AuditLog service internal wiki: "Unique identifier of the message. It is set either by the audit log client or by the user of the audit log write API in case the REST API is being used."